test/ui-test/common/testUtils.ts

```diff
@@ -25,27 +26,29 @@ export async function cleanUpNotifications(): Promise<void> {
 export async function findNotification(text: string): Promise<Notification | undefined> {
   const center = await new Workbench().openNotificationsCenter();
   const notifications = await center.getNotifications(NotificationType.Any);
-  notifications.map(async (notification) => {
-    if (notification) {
-      const message = await notification.getMessage();
-      if (message.includes(text)) {
-        return notification;
-      }
+  // eslint-disable-next-line no-restricted-syntax
+  for (const notification of notifications) {
+    // eslint-disable-next-line no-await-in-loop
+    const message = await notification.getMessage();
```

Comment:
the eslint is telling you this is bad because it can create race conditions
It is better to use `notifications.forEach()` to iterate over it I think.

Reply:
but then I will get ```Promise returned in function argument where a void return was expected.eslint@typescript-eslint/no-misused-promises```

Reply:
Anyway. how is it there is a risk of race condition? I guess that it is more likely just time consuming when it has to wait in loop to process async call for each iteration, or not? I would say rather ineffective...

Reply:
I'll merge it. I'll take another look at.